test/functionalTest/cases/2933_PATCH_offending_attributes/PATCH_offending_attributes.test

```diff
@@ -77,7 +79,7 @@ echo '03. PATCH /v2/entities/Room1/attrs, to update existing attribute (temperat
 echo '===================================================================================================================================================================='
 payload='{
   "temperature": {
-    "value": 24,
+    "value": 25,
```

Comment:
Actually not needed, but better to use a different value than the one in step 02.

NTC (informative)